docs/API_FREEZE.md

```
@@ -1,354 +1,336 @@
+   git tag vX.Y.Z
+   git push origin main --tags
+   ```
+
+5. **Publish**:
+   ```bash
+   cargo publish -p copybook-core
+   cargo publish -p copybook-codec
+   ```
+
+6. **Re-establish freeze**:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)